solver/commonSolver.py

```diff
@@ -292,11 +292,8 @@ def removeItemAt(self, locNameWeb):
         # item
         item = loc.itemName
 
-        if self.mode in ['seedless', 'race', 'debug']:
```

Comment:
this is not good. in the regular tracker you can click on a visited location to uncollect it.
and we need to respect the order in which items have been added because later we can also use 'undo last location' and calling remove for a missile will remove the first missile in the list, not the one from the location.

Reply:
why did you have to change it ? which issue did you have ?

Reply:
Ah, iirc this was during the state reset when updating from an autotracking game state.  Somewhere this function got called along the way, and it was attempting to remove the Nth collected item if we were clearing the Nth location, because it's assuming the collected items directly correspond to the visited locations in the same order.

I could add a flag of some kind when resetting during an import dump if we're reading inventory from memory?